### ISSUE TITLE
Visual Studio Code (Insiders) - Line Status Indicators Missing

### ISSUE THREAD

**Issue by Barryrowe** [wallabyjs/public#792]
### Issue description or question

In the latest version of Visual Studio Code (Insiders) the green/gray/pink/red/yellow indicator boxes on covered lines are no longer rendering. Tests still run and pass, but these boxes are no longer present.

![no-indicators-issue](https://cloud.githubusercontent.com/assets/1013426/18588562/02a89c14-7bf5-11e6-9dbe-0eb453f6c700.png)

### Wallaby.js configuration file

```javascript
var wallabyWebpack = require('wallaby-webpack');

var webpackPostprocessor = wallabyWebpack({
  entryPatterns: [
    'spec-bundle.js',
    'app/**/*spec.js'
  ]
});

module.exports = function (w) {

  return {
    files: [
      //{pattern: 'node_modules/@lge/ng2-cc/spec-bundle.js', load: false},
      {pattern: 'spec-bundle.js', load: false},      
      {pattern: 'app/**/*.ts', load: false},
      {pattern: 'app/**/*spec.ts', ignore: true},
      {pattern: "mocks/**/*.ts", load:false}      
    ],

    tests: [
      { pattern: 'app/**/*spec.ts', load: false }
    ],

    testFramework: "jasmine",

    compilers: {
      '**/*.ts': w.compilers.typeScript({
        emitDecoratorMetadata: true,
        experimentalDecorators: true
      })
    },

    postprocessor: webpackPostprocessor,

    bootstrap: function () {
      window.__moduleBundler.loadTests();
    }
  };
};
```

### Code editor or IDE name and version

Visual Studio Code v1.6.0-insider

### OS name and version

Windows 7 Professional

**Comment by Barryrowe** [comment]
Forgot to add plugin version:

Wallaby plugin 1.0.24

![plugin-version](https://cloud.githubusercontent.com/assets/1013426/18588629/3ae60f62-7bf5-11e6-8aa4-5899c54bad93.PNG)

**Comment by ArtemGovorov** [comment]
I have just tried the `Version 1.6.0-insider (1.6.0-insider) 0619ede1e469ed23fbb214eaa0377625a402ca87` version and the coverage indicators are working ok for me. 

Could you please try running wallaby on [this sample](https://github.com/wallabyjs/calculator-sample) to make sure the issue is not project specific?

Also try your project in the latest stable `Version 1.5.2 (1.5.2)` of VS Code to make sure the issue is the `insider` build specific.

**Comment by ArtemGovorov** [comment]
Also please provide
- whether or not inline messages are still working for errors/`console.logs`,
- list of other extensions you have installed ,
- output from the VS Code Dev Tools (`Ctrl + Alt + I`) (if any errors),
- check how the VS Code generated HTML look like for the gutter, here is how it should look like. There should be a `div` with `cgmr ced-TextEditorDecorationType*` classes) inside `div`s with `linenumber` attribute on lines where coverage indicators are supposed to be displayed (it how VS Code decorations should work):

<img width="753" alt="screen shot 2016-09-17 at 12 06 30 pm" src="https://cloud.githubusercontent.com/assets/979966/18605223/420f787a-7ccf-11e6-8ce7-3b5758d638f1.png">

**Comment by ArtemGovorov** [comment]
We've found the reason, something was broken in the insider build: https://github.com/Microsoft/vscode/issues/12111

**Comment by ArtemGovorov** [comment]
While the issue should also be fixed in VS Code, we have implemented a workaround in wallaby extension 1.0.25 until it gets fixed un the insiders build.

**Comment by Barryrowe** [comment]
Awesome. Always impressed with how quick you address things, even odd
edge-build items like this. Keep up the great work.
